changelog.txt

```diff
@@ -1,3 +1,295 @@
+Add: Allow loading heightmaps from command-line (#11870)
+Add: List_[scenario|heightmap] and load_[scenario|height] console commands (#11867)
+Add: Latvian Lats currency (#11691)
+Add: Horizontal scroll for script debug log (#11597)
+Add: GUI options to select sprite font and AA mode for all fonts (#11593)
+Add: Website button for basesets in Game Options window, the Game Script settings window and AI settings window (#11512)
+Add: Debug tool to draw widget outlines (#11401)
+Add: [Emscripten] Support for bootstrapping (#11109)
+Add: Hotkey to focus town / industry directory filter box (#11030)
+Add: {COLOUR} control code to allow colour changes at runtime (#10687)
+Add: Maximum number of companies allowed to the client list (#10523)
+Add: Use specific error message when vehicle cannot go to station/waypoint (#10494)
+Add: Show NewGRF name in NewGRF-created errors (#10457)
+Add: Alternative setting for right-click close window option to exclude pinned windows (#10204)
+Add: Allow autoreplace with same model vehicle (#7729)
+Add: [NewGRF] Allow inspection of road tiles and airports (#11282, #11323)
+Add: [NewGRF] Station variable 6B to get extended station id of nearby tiles (#10953)
+Add: [NewGRF] string code "9A 21" to display force from textstack (#10782)
+Add: [NewGRF] Station property 1C/1D to set name/classname (#10672)
+Add: [Script] Optional filter parameter to ScriptXXXList constructors (#11698,#11663)
+Add: [Script] AI/GS Time Mode to choose between economy (default) and calendar time (#11603)
+Add: [Script] Allow to set max loan for each company separately (#11224)
+Add: [Script] GSIndustry.GetConstructionDate() method (#11145)
+Add: [Script] Game script control of industry production level and news messages (#11141)
+Add: [Script] GSAsyncMode to set async mode of gamescript commands (#10913)
+Add: [Script] GSCompanyMode::IsValid and IsDeity, and enforce valid company/deity mode where applicable (#10536, #10529)
+Add: [Script] Allow GS to found town with random road layout (#10442)
+Add: [Script] Create own Randomizer per instance (#10349)
+Change: Better handle different GUI sizes for most windows, and squash inconsistencies between windows
+Change: Allow configuring AI slots above the current maximum number of competitors (#11961)
+Change: Forcefully enable prefixing logs with date (#11930)
+Change: Position error window closer to cursor on large screens (#11923)
+Change: Only open story-book in center when a GS does it (#11916)
+Change: Rebrand Cheats as Sandbox Options (#11874)
+Change: Make smooth-scrolling based on actual time (#11865)
+Change: Set smooth-scrolling on by default (#11860)
+Change: Disable building rail infrastructure if train build limit is zero (#11847)
+Change: Invalidate music volume when restarting music playback on Windows (#11836)
+Change: Make street lights transparent with houses (#11828)
+Change: Be more resilient against missing towns when loading TTO/TTD saves (#11786)
```

Comment:
```suggestion
```
I'd remove this, as it's mostly hypothetical. The only reason I "fixed" this, was because I was messing with loading and broke the loading. That was then loading some gibberish, which is failed on. So, not really worth to mention.